man/coap_context.txt.in

```diff
@@ -296,6 +296,8 @@ setup_server_context_pki (const char *public_cert_file,
   dtls_pki.check_cert_revocation   = 1;
   dtls_pki.allow_no_crl            = 1;
   dtls_pki.allow_expired_crl       = 1;
+  dtls_pki.allow_bad_md_hash       = 1;
```

Comment:
I think that it would be advisable to have the less secure options `0`.

Reply:
Correct - updated.